kinto/config/kinto.tpl

```diff
@@ -92,21 +92,13 @@ multiauth.policies = basicauth
 # Set `kinto.includes` to `kinto.plugins.accounts`
 # Enable authenticated policy.
 # Set `multiauth.policies` to `account`
-# multiauth.policy.account.use = kinto.plugins.accounts.authentication.AccountsAuthenticationPolicy
+multiauth.policy.account.use = kinto.plugins.accounts.AccountsPolicy
 # Allow anyone to create accounts.
-# kinto.account_create_principals = system.Everyone
+kinto.account_create_principals = system.Everyone
 # Set user 'account:admin' as the administrator.
-# kinto.account_write_principals = account:admin
-# kinto.account_read_principals = account:admin
-#
-# Kinto-portier authentication
-# https://github.com/Kinto/kinto-portier
-# Set `multiauth.policies` to `portier`
-# multiauth.policy.portier.use = kinto_portier.authentication.PortierOAuthAuthenticationPolicy
-# kinto.portier.broker_url = https://broker.portier.io
-# kinto.portier.webapp.authorized_domains = *.github.io
-# kinto.portier.cache_ttl_seconds = 300
-# kinto.portier.state.ttl_seconds = 3600
+kinto.account_write_principals = account:admin
```

Comment:
Should we enforce admin here as the default root user? I believe it would help people but it seems a bit brittle regarding security.

Reply:
`account:admin` suggests that the admin account should exists. Also I would rather encourage people to use an email address as username because it simplifies the use of the sharing capabilities.

Reply:
> account:admin suggests that the admin account should exists.

Yes maybe there are some place in the getting started docs that don't mention creating the admin. I added mentions in the install docs

> Also I would rather encourage people to use an email address as username because it simplifies the use of the sharing capabilities (although for the later we might need to verify the address)

We could have a regex for account ids, whose default would be an email adress for example. But please open another issue, I don't want to let this rot ;)

Reply:
> But please open another issue, I don't want to let this rot

I wasn't suggesting that we should make any changes or enforce the email address there, I was just suggesting to use account:``your.email@host.tld`` rather than ``account:admin``

Reply:
For the sake of simplicity I think I prefer `account:admin` :/